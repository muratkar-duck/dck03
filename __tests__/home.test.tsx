import React from 'react';
import { renderToStaticMarkup } from 'react-dom/server';

jest.mock('react-simple-typewriter', () => ({
  Typewriter: ({ words }: { words: string[] }) => <span>{words[0]}</span>,
}));

describe('Home page', () => {
  it('matches snapshot', async () => {
    const HomePage = (await import('@/app/page')).default;
    const html = renderToStaticMarkup(<HomePage />);

    expect(html).toMatchSnapshot();
  });
});
