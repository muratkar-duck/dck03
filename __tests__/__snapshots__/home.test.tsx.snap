// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Home page matches snapshot 1`] = `"<div class="space-y-16"><section class="text-center"><h1 class="text-4xl font-bold mb-4">Senaryonuz DÃ¼nyaya UlaÅŸsÄ±n</h1><p class="text-lg max-w-2xl mx-auto font-sans text-[#7a5c36] overflow-hidden" style="min-height:4rem;max-height:4rem"><span>ducktylo, senaristlerle yapÄ±mcÄ±larÄ± bir araya getiren yenilikÃ§i bir platformdur.</span><span class="blinking-cursor">|</span></p><div class="mt-8 flex justify-center gap-4 flex-wrap"><a href="/auth/sign-up-writer" class="btn btn-primary">Senarist Olarak KatÄ±l</a><a href="/auth/sign-up-producer" class="btn btn-secondary">YapÄ±mcÄ± Olarak KatÄ±l</a><a href="/auth/sign-in" class="btn btn-warning">GiriÅŸ Yap</a></div></section><section><h2 class="text-2xl font-semibold mb-6 text-center">ğŸš€ Ã–zellikler</h2><div class="grid md:grid-cols-3 gap-6"><div class="card"><h3 class="text-lg font-semibold mb-2">Senaryo YÃ¼kleme</h3><p>SenaryolarÄ±nÄ±zÄ± yÃ¼kleyin, fiyatlandÄ±rÄ±n, yayÄ±nlayÄ±n. Her ÅŸey kontrolÃ¼nÃ¼zde.</p></div><div class="card"><h3 class="text-lg font-semibold mb-2">YapÄ±mcÄ± Filtrelemesi</h3><p>YapÄ±mcÄ±lar tÃ¼r, sÃ¼re ve bÃ¼tÃ§eye gÃ¶re senaryolara ulaÅŸÄ±r.</p></div><div class="card"><h3 class="text-lg font-semibold mb-2">GÃ¼venli Ã–deme</h3><p>Komisyon sistemiyle korunan adil Ã¶deme altyapÄ±sÄ±.</p></div></div></section></div>"`;
